packages/motion-start/src/components/Reorder/__tests__/ReorderGroupInvariantFixture.svelte

```
@@ -0,0 +1,13 @@
+<script lang="ts">
+import Group from '../Group.svelte';
+
+let values = $state<string[] | undefined>(['alpha']);
```

Biome (2.5.5)

[error] This variable values is unused.

(lint/correctness/noUnusedVariables)